provider/testdata/data/baked/Cargo.toml

```diff
@@ -1,12 +1,26 @@
 [package]
 name = "baked"
 version = "1.0.0"
+description = "Pre-built test data for ICU4X"
+authors = ["The ICU4X Project Developers"]
 edition = "2021"
-include = ["src/**/*.rs", "Cargo.toml"]
+repository = "https://github.com/unicode-org/icu4x"
+license-file = "LICENSE"
+categories = ["internationalization"]
```

Comment:
Hmm I didn't realize that we'd have to publish this on its own if we make it a separate crate, and I don't think we should do that. Maybe we can include its files in `icu_testdata` directly. Or is there some way to have unlisted crates on crates.io? @sffc wdyt?

Reply:
I think at the bare minimum it should be called `icu_baked_testdata` or something

But I'd prefer to not publish it too.

Reply:
I'll rename it here. I agree, we shouldn't publish it separately, but that's a larger change.